src/components/MoreInfoPage/index.test.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import MoreInfoPage from '.';
+
+
+describe('MoreInfoPage', () => {
+  let wrapper;
+
+  it('should match the snapshot', () => {
+    wrapper = shallow(<MoreInfoPage />);
+
+    expect(wrapper).toMatchSnapshot();
+  });
+});
```

Comment:
Look at all this TDD! I think it helps to not want to make CI angry...or @CSheesley 😅